### PR TITLE
Add zeromq package.

### PR DESCRIPTION
**Pull request by dlorenc**
This is required for nginx.

Signed-off-by: Dan Lorenc <dlorenc@chainguard.dev>

### CODE REVIEW

**Comment by dlorenc** [comment]
Depends on #320

**Comment by dlorenc** [comment]
Rebased, should be good now.

**Review by rawlingsj — DISMISSED** [review]

File: zeromq.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          license: LGPL-3.0-or-later with exceptions
+environment:
+    contents:
+        packages:
+            - busybox
+            - ca-certificates-bundle
+            - build-base
+            - automake
+            - autoconf
+            - autoconf
```
Comment (imjasonh) [inline zeromq.yaml:22]: Duplicate

Comment (dlorenc) [reply zeromq.yaml]: Fixed!

**Review by rawlingsj — APPROVED** [review]

**Review by luhring — APPROVED** [review]

**Pull request merged**